pom.xml

```
@@ -0,0 +1,120 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.springframework.boot</groupId>
+    <artifactId>spring-boot-starter-parent</artifactId>
+    <version>3.3.5</version>
+    <relativePath/> <!-- lookup parent from repository -->
+  </parent>
+  <groupId>org.anastasiiapanchenko</groupId>
+  <artifactId>SpringBoot</artifactId>
+  <version>1.0-SNAPSHOT</version>
+  <name>Archetype - SpringBoot</name>
+  <url>http://maven.apache.org</url>
+<properties>
+  <java.version>23</java.version>
```

Better use LTS versions in production
```suggestion
  <java.version>21</java.version>
```